res/controllers/Pioneer-DDJ-SB-scripts.js

```diff
@@ -28,12 +28,22 @@ PioneerDDJSB.jogwheelSensitivity = 1.0;
 // Set to 1 to disable jogwheel sensitivity increase when holding shift.
 PioneerDDJSB.jogwheelShiftMultiplier = 20;
 
+// If true, releasing browser knob jumps forward to jumpPreviewPosition.
+PioneerDDJSB.jumpPreviewEnabled = true;
+// Position in the track to jump to. 0 is the beginning of the track and 1 is the end.
+PioneerDDJSB.jumpPreviewPosition = 0.5;
+
 ///////////////////////////////////////////////////////////////
 //                      INIT & SHUTDOWN                      //
 ///////////////////////////////////////////////////////////////
 
 
-PioneerDDJSB.init = function (id) {
+PioneerDDJSB.init = function(id) {
+
+    PioneerDDJSB.effectUnit = [];
+    PioneerDDJSB.effectUnit[1] = new PioneerDDJSB.EffectUnit(1);
+    PioneerDDJSB.effectUnit[2] = new PioneerDDJSB.EffectUnit(2);
```

Comment:
wouldn't the (components.EffectUnit](https://github.com/mixxxdj/mixxx/wiki/Components%20JS#effectunit) suffice?

Reply:
I wasn't aware of that component, let me dig into it!

Reply:
Ah yeah, that makes sense. then the custom implementation is probably the better choice, I'll have a look asap.